### PR TITLE
[NayNay] CLI/TUI File Restructure

### LINKED ISSUE #107: [anchor#102 sub issue] start with a better file structure
WIP notes: https://hackmd.io/@naynay/BypjxFrYC/edit

- [x] #208
- [x] #209
- [x] #210
- [ ] #211
- [x] #212

### PR DESCRIPTION
**Pull request by frankiebee**
This PR is for the restructure of the code base to allow it to be more flexible.

- closes #107

### CODE REVIEW

**Review by frankiebee — COMMENTED** [review]
`inquirer` is in the command files which means for programmatic cli we will be loading it and not using it. Lets find a way that we can leave it out of `command.ts` files

**Comment by mixmix** [comment]
yeah that's an artifact of e.g. that `initializeEntropy` being written tui-first. I refactored enough to isolate it some, and wrote a warning that this needed to change. Excited we're at that point!

File: src/cli.ts [hunk 1]
```diff
@@ -58,8 +104,20 @@ program
...
+      // balance includes an address argument, use that address to instantiate entropy
+      // can keep the conditional to check for length of args, and use the first index since it is our pattern to have the address as the first argument
+      if (RESTRUCTURED_COMMANDS.includes(actionCommand.name()) && actionCommand.args.length) {
+        await loadEntropy(actionCommand.args[0], actionCommand.opts().endpoint, actionCommand.opts().password)
+      } else {
+        // if address is not an argument, use the address from the option
+        await loadEntropy(actionCommand.opts().account, actionCommand.opts().endpoint, actionCommand.opts().password)
+      }
+    }
+  })
```
Comment (rh0delta) [inline src/cli.ts:118]: @mixmix this is the abstraction of entropy I had implemented, where we use the pre action hook to initialize/load entropy

Comment (mixmix) [reply src/cli.ts:118]: @rh0delta  cool!

Why would the account address ever be `args[0]`? I would have expected it to be an option... also is this stuff gonna change when we get into CLI sessions?

**Pull request closed without merging**